templates/defaultfilters/stringfilters.cpp

```diff
@@ -584,3 +588,31 @@ QVariant TruncateCharsFilter::doFilter(const QVariant &input, const QVariant &ar
     retString.append(QStringLiteral("..."));
     return retString;
 }
+
+QVariant JsonScriptFilter::doFilter(const QVariant &input, const QVariant &argument, bool autoescape) const
+{
+    Q_UNUSED(autoescape)
+    const QString arg = escape(getSafeString(argument));
+
+    QJsonDocument json;
+
+    if (input.canConvert<QJsonDocument>()) {
+        json = input.toJsonDocument();
+    } else if (input.canConvert<QJsonObject>()) {
+        json.setObject(input.toJsonObject());
+    } else if (input.canConvert<QJsonArray>()) {
+        json.setArray(input.toJsonArray());
+    } else {
+        qWarning("%s", "Can not convert input data into QJsonObject or QJSonArray.");
+        return QVariant();
+    }
+
+    QString jsonString = QString::fromUtf8(json.toJson(QJsonDocument::Compact));
+    jsonString.replace(QLatin1Char('<'), QStringLiteral("\\\\u003C"));
```

Comment:
This part could be done like the escape() function which is a bit faster since it iterates the string a single time.

Reply:
I meant this way of escaping
https://github.com/cutelyst/cutelee/blob/master/templates/lib/outputstream.cpp#L33 only that since you have a non-const string it would be nicer to replace inline.

Reply:
Ah, will change it.